packages/terra-props-table/src/PropsTable.jsx

```diff
@@ -1,12 +1,12 @@
 /* eslint-disable import/no-extraneous-dependencies */
 import React from 'react';
 import PropTypes from 'prop-types';
-import { parse } from 'react-docgen';
```

Comment:
You are still using the parse method below. Could we keep this import and add the resolver to keep our bundle size smaller?

Reply:
@emilyrohrbough Thanks Emily, when i imported [findAllComponentDefinitions](https://github.com/reactjs/react-docgen/blob/master/src/resolver/index.js#L18) i couldn't get resolver to work. I will take a look again.

Reply:
@m-samkc  I think you would want to import the resolver:

```
import { parse,  resolver} from 'react-docgen';
```

Then below:
```
const componentMetaData =parse(src, resolver.findAllComponentDefinitions);
```